tensorboard/webapp/app_routing/testing.ts

```diff
@@ -35,13 +34,26 @@ export function buildCompareRoute(
 ): Route {
   return {
     routeKind: RouteKind.COMPARE_EXPERIMENT,
-    pathname: '/campare',
+    pathname: '/compare',
     params: {experimentIds: aliasAndExperimentIds.join(',')},
     queryParams: [],
     ...routeOverride,
   };
 }
 
+export function buildExperimentRoute(
+  experimentId: string,
+  routeOverride: Partial<Route> = {}
```

Comment:
The input is a bit confusing to me. I won't know to pass experiment id from the name. Also the level of usage for the second param is different from the first one. Would you consider combining the two params to be more general input?
for example  `buildExperimentRoute({params: {experimentId: 'foo'}})`?

Or if the second param is not used, can we rename to buildExperimentRouteFromId('foo')?

Reply:
Done. Changed to buildExperimentRouteFromId.